core/src/main/java/org/infinispan/manager/DefaultCacheManager.java

```diff
@@ -1075,14 +1075,15 @@ public Health getHealth() {
 
    @Override
    public ClusterExecutor executor() {
+      authzHelper.checkPermission(AuthorizationPermission.EXEC);
```

Comment:
`master` requires `ADMIN`, we should sync them

Reply:
I believe master should be `EXEC`, I think `ADMIN` was a mistake. I was planning on updating this after this PR was integrated.

Reply:
I suggest changing the master PR right now. If you want to document the change you can create a separate JIRA/commit, but after I added the comment I read the `EXEC` javadoc and it's pretty clear that it applies (although we no longer have the distributed executor or map/reduce).

Reply:
I could do it right now, but if something in here changes I would also have to change it on master. I wanted to add it as soon as this was integrated, so I know what changes have to be duplicated.

Reply:
I actually wanted to merge `master` first, but I added the comments on this PR because you added the permission check here and it was already there in master :)